### PR TITLE
feat: Smart Items simplified view

### PR DESCRIPTION
**Pull request by cyaiox**
This PR adds a new simplified UI for specific smart items, implementing a new component: `Config` to store the layout, associated triggers, and a default pointer event.

- Doors:
image

- Chests:
image

- Buttons:
image

- Levers:
image

- Platforms:
image

- Enable Advance View
image
image

- Return to Basic View
image
image

### CODE REVIEW

**Comment by cazala** [comment]
I notice some weird behavior on the NFT Portrait. When I change the color the frame get resetted to None, nd when I change the tokenId sometimes it makes the frame change to its previous value (which might not be None).

https://github.com/decentraland/js-sdk-toolchain/assets/2781777/c9d47304-bacb-4d24-bb90-80cc6dc7261b

**Comment by cazala** [comment]
Is it okay that the "Lever" component can be removed? This removes the Config component, so it changes the view to the Advanced View but without warning, and it cannot be undone.

Screenshot 2024-03-29 at 10 57 30 AM

File: packages/@dcl/inspector/src/components/EntityInspector/EntityInspector.tsx [hunk 1]
```diff
@@ -26,46 +27,133 @@ import { PointerEventsInspector } from './PointerEventsInspector'
...
+        name: sdk.components.Animator.componentName,
+        component: AnimatorInspector
+      },
+      {
+        sdkComponent: sdk.components.PointerEvents,
+        name: sdk.components.PointerEvents.componentName,
+        component: PointerEventsInspector
+      },
+      {
+        sdkComponent: sdk.components.SyncComponents,
```
[the remark this answers is not in the document]
Comment (cyaiox) [reply packages/@dcl/inspector/src/components/EntityInspector/EntityInspector.tsx:129]: I forgot to remove this prop. It's not used.. removing it

**Comment by cyaiox** [comment]
> Is it okay that the "Lever" component can be removed? This removes the Config component, so it changes the view to the Advanced View but without warning, and it cannot be undone.
> 
> Screenshot 2024-03-29 at 10 57 30 AM

good catch! shouldn't be removed

File: packages/@dcl/inspector/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "@dcl/inspector",
   "version": "0.1.0",
   "dependencies": {
-    "@dcl/asset-packs": "1.15.2",
+    "@dcl/asset-packs": "1.15.3-20240404141248.commit-55a9931",
```
Comment (cazala) [inline packages/@dcl/inspector/package.json:5]: We should release `@dcl/asset-packs` and update here before merging

**Pull request merged**